LLMonFHIR/FHIR Display/FHIRResourcesView.swift

```diff
@@ -51,9 +59,39 @@ struct FHIRResourcesView: View {
                 .sheet(isPresented: $showSettings) {
                     SettingsView()
                 }
+                .sheet(isPresented: $showMultipleResourcesChat) {
+                    resourceChatView
+                }
+                .task {
+                    allResourcesArray = await fhirStandard.resources
+                    await interpretMultipleResources()
+                }
                 .navigationTitle("FHIR_RESOURCES_TITLE")
         }
     }
+    
+    @ViewBuilder
+    private var resourceChatView: some View {
+        ResourceChatView(
+            chat: fhirMultipleResourceInterpreter.chat(resources: allResourcesArray),
+            title: "All FHIR Resources"
+         )
```

Comment:
Please indent elements by 4 spaces.
```suggestion
        )
```